game.js

```diff
@@ -0,0 +1,49 @@
+var userName = prompt("What is your name?");
+console.log("User input for userName: " + userName);
```

Comment:
This is fine for now, but generally comments and console logs are used for debugging code locally, and shouldn't make it to production code (GitHub).